README.md

```diff
@@ -1 +1,152 @@
-# CrystFEL-Jupyter-utilities
\ No newline at end of file
+# Description:
+
+
+H5see is a project with graphical tools from CrystFEL implemented in Python.
+The project is intended to work as a wrapper for CrystFEL tools when it is not 
+possible to use a graphical environment but graphical (interactive) features 
+are needed. The tools were written to resemble original CrystFEL tools while 
+at the same time feature some of the requests from CrystFEL users. 
+In particular following tools are available:
+hdfsee
+check-peak-detection.sh (script)
```

Comment:
```suggestion
 * check-peak-detection.sh (script)
```

Reply:
Done in: c717bd7f25d2cb8c2ac5dcb1bfe7c03873775530